utilities/check-db-schema.sh

```diff
@@ -2,7 +2,7 @@
 
 # Stop exiting and return non-zero error code if an error occurs
 set -ue
-
+export POSTGRESQL_DATABASE="postgres"
```

Comment:
As above, if the environment variable is already set by the user calling script, we should use the existing value, otherwise we should default to `postgres`